src/navigation/app/app-navigation.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import { AppRoute } from '../../common/enums/navigation';
+import HomeScreen from '../../screens/Home';
+import ExploreScreen from '../../screens/Explore/ExploreScreen';
+import OpportunitiesScreen from '../../screens/Opportunities/OpportunitiesScreen';
+import OKRScreen from '../../screens/OKR/OKRScreen';
+import ProfileScreen from '../../screens/Profile/ProfileScreen';
```

Comment:
I'd suggest we stick to following structure:
```
src/
-- screens/
--- Home/
---- index.tsx
---- styles.ts
--- Explore/
---- index.tsx
```

Let's discuss.

Reply:
ok, i'll change